input/pagecontent/index.md

```diff
@@ -1,14 +1,17 @@
 ### Introduction
-This implementation guide is provided to support the use of HL7<sup>&reg;</sup> FHIR<sup>&reg;&copy;</sup> in an Australian context.
+This implementation guide is provided to support the use of HL7<sup>&reg;</sup> FHIR<sup>&reg;&copy;</sup> in an Australian context.  
 
-The AU Base implementation guide presents Australian use concepts defined via FHIR processable artefacts; these are collaborative outputs with agreed approaches to varied kinds of healthcare related information based on the core FHIR R4 specification as base profiles that can be further constrained in a separate implementation guide for a particular usage.
+The AU Base implementation guide defines Australian realm concepts (e.g. Medicare card number) as an additional set of options to what is available in the core FHIR standard, including extensions, terminology and identifiers.  It provides base profiles to inform a reader of which added concepts are considered relevant to a particular resource type, and these base profiles may be further constrained in a separate implementation guide for a particular usage.  For this reason, the AU Base implementation guide does not apply cardinality constraints or required binding strengths to added concepts (except in rare circumstances), utilise must support flags or recommend or mandate any particular resource, element or interactions.
 
 This guide alone does not constrain profiles sufficiently to ensure implementation use cases can be met. 
 
 For an implementation guide that has basic constraints to support many uses it is recommended that the AU Core implementation guide is used.
 The implementable AU Core specification references and uses AU Base profiles, this constrains elements further and defines an expected level of interface interaction support around profiles.
 If an implementation is AU Core conformant this allows a level of core capability to be assumed when using.
 
+### Dependencies
+
+{% include dependency-table.xhtml %}
 
 ### How to Read this Guide
```

Comment:
The How to Read section is inconsistent with the submitted PR for AU Core, e.g. different descriptions of the Downloads page. One has a support page, one doesn't.

Make them consistent, then resubmit the PRs, also submit the equivalent AU eRequesting PR. Review and approval of all three will occur in parallel.

Reply:
Addressed in commit below.